frontend/compositions/enrollment-response/enrollment-response.tsx

```diff
@@ -0,0 +1,101 @@
+import React from 'react'
+import Link from 'next/link'
+import styles from './enrollment-response.module.css'
+import { Logo, ExternalLink } from '../../shared-components'
+import { AppRoutes, LogoSizes } from '../../models'
+
+interface EnrollmentResponseProps { isSuccess?: boolean }
+
+interface EnrollmentErrorText {
+  statusMessage: string,
+  returnText: string,
+  returnPath: string
+}
+
+export enum EnrollmentTypes {
+  VIEWER = 'viewer',
+  PASSPORT = 'passport'
+}
+
+const enrollmentMessage: { [key in EnrollmentTypes]: EnrollmentErrorText} = {
```

Comment:
I think this & the above enums/interfaces (except for props) should probably live in our Models directory!

Reply:
That makes sense, but I don't understand where they should live, exactly. Is there some kind of method to how the Models directory is organized? I couldn't find any existing types that would naturally include "Viewer" & "Passport", even though they seem like obvious reusable parts of something

Reply:
Hmm I'm not sure I totally follow - currently the models directory is just flat with a funnel `index.tsx` file to export everything in the directory (so that subsequent imports can be consolidated)

I'd totally be open to reorganizing that if that would help! I had just heard from others on the team that keeping a relatively flat structure was a priority for us